viewer/src/services/navigation.ts

```diff
@@ -40,8 +40,8 @@ export default class Navigation {
   }
 
   // Checks if the type of an item matches
-  public static checkType(item: Gallery.Item | null, type: Gallery.ItemType): boolean {
-    return item?.properties.type === type ?? false;
+  public static checkType(item: Gallery.Item | null, type: Gallery.ItemType | null): boolean {
+    return item?.properties.type === type;
```

Comment:
`item?` may return undefined, and `undefined === null` returns false.
```suggestion
    return (item?.properties.type ?? null) === type;
```